### PR TITLE
NAS-110544 / 22.02-RC.2 / Added 'xattr' with different messages/labels for PUSH/PULL

### PR DESCRIPTION
**Pull request by RehanY147**
NAS-110544

### CODE REVIEW

File: src/app/pages/data-protection/cloudsync/cloudsync-form/cloudsync-form.component.ts [hunk 1]
```diff
@@ -687,7 +694,18 @@ export class CloudsyncFormComponent implements FormConfiguration {
     // When user interacts with direction dropdown, change transfer_mode to COPY
     this.formGroup
       .get('direction')
-      .valueChanges.pipe(filter(() => this.formGroup.get('transfer_mode').value !== TransferMode.Copy))
+      .valueChanges
+      .pipe(tap((direction: string) => {
+        const xattr_fc = entityForm.fieldConfig.find((config: any) => config.name === 'xattr');
```
Comment (undsoft) [inline src/app/pages/data-protection/cloudsync/cloudsync-form/cloudsync-form.component.ts:699]: `xattr_fc` => `xattrFc` or even `xAttrControl`

`config` is probably of type `FieldConfig`

**Comment by undsoft** [comment]
Is this something that was not implemented by middleware? I just get 'Field not expected' when I try to submit form.

**Review by undsoft — CHANGES_REQUESTED** [review]
Is this something that was not implemented by middleware? I just get 'Field not expected' when I try to submit form.

**Comment by RehanY147** [comment]
Yes. Seems like the related middleware ticket is on the back burner for now. Labeling this blocked.

**Pull request closed without merging**